server/server_test.go

```diff
@@ -1322,12 +1324,28 @@ func startTestServer(config string) error {
 	if err != nil {
 		return err
 	}
+
 	go func() {
 		err := server.Start()
 		if err != nil {
 			return
 		}
 	}()
+
+	retry := 3
```

Comment:
This constant is in the middle of the code. It's better to move it to the const section at the top of the source code.